### DISCUSSION TITLE
事务消息支持分支事务吗？

### DISCUSSION THREAD

**Discussion by lanjiexin** [apache/rocketmq#6835] [Q&A] [answered]
这边有个场景：有三个事务消息，如果其中有一个半消息被取消了，需要自动取消另外两个，也就是说需要等这三个半消息都被同意后再统一提交。有办法实现吗？

**Comment by fuyou001** [comment, marked as answer]
不支持这种场景，当前支持消息和或其它事务源的事务，不支持多个消息间的事务。

**Comment by ferrirW** [comment]
@lanjiexin 可以稍微展开下具体是什么场景吗，我可以看看开发支持一下这种需求

**Comment by mz0113** [comment]
这简单啊 , 你三个消息回查时候用相同的txId回查不就好了 .
